src/silx/app/view/CustomPlotSelectionWindow.py

```diff
@@ -21,24 +21,28 @@
 # THE SOFTWARE.
 # ###########################################################################*/
 
-""""""
+"""Custom plot selection window for selecting 1D datasets to plot."""
```

Comment:
So that most of the typing syntax (e.g. use of `|` with types) is supported with Python v3.8, this is needed:

```suggestion
from __future__ import annotations
"""Custom plot selection window for selecting 1D datasets to plot."""
```